### PR TITLE
add rule for cz-pulsar

### PR DESCRIPTION
**Pull request by mira-miracoli**
Followup for #787 
@martindemko

### CODE REVIEW

File: files/galaxy/tpv/tool_defaults.yml [hunk 1]
```diff
@@ -41,6 +41,8 @@ tools:
                     pulsar_tag = user_preferences[data_item]
           new_tag = {'require':[pulsar_tag]} if pulsar_tag else {}
           entity.tpv_tags = entity.tpv_tags.combine(TagSetManager.from_dict(new_tag))
+          if pulsar_tag == 'cz-pulsar' and 'alphafold' in tool.id:
```
[the remark this answers is not in the document]
Comment (mira-miracoli) [reply files/galaxy/tpv/tool_defaults.yml:44]: no it is inside python code, it worked when testing with dryrun

[the remark this answers is not in the document]
Comment (sanjaysrikakulam) [reply files/galaxy/tpv/tool_defaults.yml:44]: Ok, I thought it would be a separate rule.

**Review by sanjaysrikakulam — APPROVED** [review]

**Comment by martindemko** [comment]
Do I understand it correctly that our pulsar should be able to run any version of Alphafold2 with GPUs?

**Comment by mira-miracoli** [comment]
If you want I can limit it to alphafold 2.3.1

**Comment by kysrpex** [comment]
> Do I understand it correctly that our pulsar should be able to run any version of Alphafold2 with GPUs?

Would you like to be able to run all versions or just one or a few?

**Review by kysrpex — APPROVED** [review]

**Comment by martindemko** [comment]
It's still in the testing phase, so far I'm good with the 2.3.1 but for the future, I can manage versions on my own simply by selecting the one I want, correct? In that case, I don't see the point of limiting it. What I wonder more about now is if I will have to manually select pulsar (AKA runner) between the tools, as one runner should be oriented only for GPU computing and the other one for non-GPU computing. Thanks

**Comment by bgruening** [comment]
Let's start with the latest version. The DB handling of alphafold is not easy and a science on its own ;-)

@martindemko we should be able to run jobs on your end with GPU and on our with CPU. The tool supports an ENV variable that we simply set and this then determines the GPU/CPU

**Comment by kysrpex** [comment]
> Let's start with the latest version. The DB handling of alphafold is not easy and a science on its own ;-)
> 
> @martindemko we should be able to run jobs on your end with GPU and on our with CPU. The tool supports an ENV variable that we simply set and this then determines the GPU/CPU

@bgruening I think what we merged does not let him make use of the GPU because this rule overwrites the environment variable.

```
- if: helpers.tool_version_gte(tool, '2.3.1+galaxy2')
        params:
          singularity_run_extra_arguments: "--env ALPHAFOLD_DB=/data/db/databases/alphafold_databases,ALPHAFOLD_USE_GPU=False"
```

**Comment by bgruening** [comment]
but we could make this env also dependent on the availability of GPUs correct?

**Comment by kysrpex** [comment]
> but we could make this env also dependent on the availability of GPUs correct?

That would be the wisest option.

**Comment by martindemko** [comment]
Hi guys, from your discussion, I assumed it wouldn't work with GPU, so is there anything new since then?

**Pull request merged**